movies_import/utils.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+import requests
+from django.contrib import messages
+from django.core.exceptions import ValidationError
+from zipfile import ZipFile
+from django.db import IntegrityError
+from cinema.settings import MEDIA_ROOT
+import csv
+from pathlib import Path
+from django.core.files import File
+from movies.models import Movie
+
+
+def validate_zipfile(zip_file):
```

Comment:
I think this function is not used anymore (from what I see from Github - in case it is, pardon me and ignore this comm). Try to make a habit of removing unused code. It will be of great help later on